cmd/api-errors.go

```diff
@@ -32,11 +32,11 @@ import (
 	"github.com/minio/minio/cmd/crypto"
 	"github.com/minio/minio/cmd/logger"
 	"github.com/minio/minio/pkg/auth"
+	"github.com/minio/minio/pkg/bucket/objectlock"
+	"github.com/minio/minio/pkg/bucket/policy"
+	"github.com/minio/minio/pkg/bucket/tagging"
```

Comment:
Tagging is technically objectTagging. Can we name it as `pkg/object/tagging`?

Reply:
Objectlock should also be under pkg/object/lock since locking is a object level feature

Reply:
Yes but I prefer to keep bucket as a primary top level directory. It is not a representation of API but more about how these features are used for objects or bucket on a bucket.